### PR TITLE
Solution for Exercise 05 from Chapter 01 in Baby Rudin

### PR DESCRIPTION
**Pull request by gapry**
This pull request provides the solution for Exercise 05, Chapter 01, from the third edition of Principles of Mathematical Analysis by Walter Rudin.

### CODE REVIEW

File: Books/BabyRudin/Chapter01/ex05.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+\subsection*{Exercise 05}
+\begin{flushleft}
+$\text{(1) } \exists\ \alpha \in R \ni  x \geq  \alpha \ \forall  x \in  A$ \\
+$\text{(2) } \exists\ \alpha \in R \ni -x \leq -\alpha \ \forall -x \in -A$ \\
```
Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter01/ex05.tex:4]: For (2), you use the same symbol $`\alpha`$ as in (1), but do you mean a different $`\alpha`$ here? The symbol usage is confusing here.

Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter01/ex05.tex:4]: You cannot use `\forall -x`, to be a syntactically correct logic statement, `\forall` must be followed by a variable but not an expression.

File: Books/BabyRudin/Chapter01/ex05.tex [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+\end{flushleft}
+
+\begin{flushleft}
+(1) $\implies  \alpha = \inf( A)$ \\
```
Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter01/ex05.tex:8]: (1) only implies the $`\alpha`$ is a lower bound, it is not necessarily a greatest lower bound, similar to the below.

File: Books/BabyRudin/Chapter01/ex05.tex [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+(2) $\implies -\alpha = \sup(-A)$
+\end{flushleft}
+
+\begin{flushleft}
+Obviously, $a = -(-a) \implies \inf(A) = -\sup(-A)$
```
Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter01/ex05.tex:13]: What is $`a`$? Do you mean $`\alpha`$? Which $`\alpha`$, from (1) or from (2)? The confusing usage of symbol above make this hard to reason here.

File: Books/BabyRudin/Chapter01/ex05.gapry.tex [hunk 4]
```diff
@@ -0,0 +1,11 @@
+\subsection*{Exercise 05 (Gapry)}
+
+\begin{flushleft}
+Since $A \subset R$, and $A$ is bounded below, we know that $\inf A$ exists.
+Let's claim $\alpha = \inf A$, so that $A = \{x \in R\ |\ x \ge \alpha \}$.
```
Comment (ceciliachan1979) [inline Books/BabyRudin/Chapter01/ex05.gapry.tex:5]: Let say $` A = \{1,2,3,4,5\} `$, it is bounded below and therefore $`\inf(A) = 1`$ exists.

This line says $`A = \{x \in R\ |\ x \ge 1 \}`$ and therefore it includes 1.1, which is not correct.

**Comment by ceciliachan1979** [comment]
Old PR note, rebased to main, this solution still have a major flaw.

**Comment by ceciliachan1979** [comment]
@E7-87-83, logic 上我搵唔到問題了，但硬係覺得 presentation 好怪，你可唔可以指點一下？

**Comment by E7-87-83** [comment]
".... we know that for all y ∈ −A, y ≤ sup(−A) is true."
呢句需要證明。

**Comment by E7-87-83** [comment]
哦……

不如咁改寫 "Let’s claim y = −x. By definition of sup, for all y ∈ −A, y ≤ sup(−A) is true."

**Pull request merged**